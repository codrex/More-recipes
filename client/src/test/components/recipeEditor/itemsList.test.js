import React from 'react';
import toJson from 'enzyme-to-json';
import { Provider } from 'react-redux';
import configureStore from '../../../store/configStore';
import ItemsList from '../../../components/pages/RecipeEditor/AddItems/ItemsList';

const props = {
  deleteItem: jest.fn(),
  editItem: jest.fn(),
  items: [{}],
  name: 'directions',
  directions: true,
  uniquenessChecker: jest.fn()
};

describe('Items list component ', () => {
  test('render as expected', () => {
    const component = shallow(
      <Provider store={configureStore()}>
        <ItemsList {...props} />
      </Provider>);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
