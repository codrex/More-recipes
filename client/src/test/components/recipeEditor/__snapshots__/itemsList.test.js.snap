// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items list component  expected to match snapshot 1`] = `
<ItemsList
  deleteItem={[Function]}
  directions={true}
  editItem={[Function]}
  ingredients={false}
  items={
    Array [
      Object {},
    ]
  }
  name="directions"
  uniquenessChecker={[Function]}
/>
`;
